### PR TITLE
fix ci to work with upstream

### PR DESCRIPTION
**Pull request by kshahat**
https://github.com/spotify/bigtable-autoscaler/pull/108 broke the way we do things, but it fixed some previous issues. Updating CircleCI to be compliant with the new way of building images.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,33 +2,36 @@ version: 2
 jobs:
   build:
     machine:
-      image: ubuntu-1604:202004-01
+      image: ubuntu-1604:202004-01
```
Comment (dangermike) [inline .circleci/config.yml]: trailing whitespace

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,33 +2,36 @@ version: 2
...
     machine:
-      image: ubuntu-1604:202004-01
+      image: ubuntu-1604:202004-01 
     working_directory: ~/bigtable-autoscaler
     steps:
       - checkout
+      - run: 
+          name: Install Maven
+          command:
+            sudo apt-get update && sudo apt-get install maven
```
Comment (dangermike) [inline .circleci/config.yml:12]: I would have thought you need a `-y` on the install. guess not?

Comment (kshahat) [reply .circleci/config.yml:12]: what is -y?

Comment (kshahat) [reply .circleci/config.yml:12]: ah found it

Comment (kshahat) [reply .circleci/config.yml:12]: I can add to future proof lol?

**Review by bennorton767 — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (dangermike) [reply .circleci/config.yml:12]: if you want to future-proof, replace `apt-get` with `apt`. Honestly, I don't really care

**Review by dangermike — APPROVED** [review]

**Pull request merged**